apps/spruce/src/pages/task/metadata/index.tsx

```diff
@@ -101,296 +100,341 @@ export const Metadata: React.FC<Props> = ({ error, loading, task, taskId }) => {
+          <OOMTrackerMessage>
+            Out of Memory Kill detected{" "}
+            {oomTracker.pids ? `(PIDs: ${oomTracker.pids.join(", ")})` : ""}
+          </OOMTrackerMessage>
+        )}
+        {resetWhenFinished && (
+          <MetadataItem data-cy="reset-when-finished">
+            This task will restart when all of the tasks in its task group have
+            finished running.
+          </MetadataItem>
+        )}
+        {taskTrace && startTime && finishTime && (
+          <MetadataItem>
+            <HoneycombLinkContainer>
+              <StyledLink
+                data-cy="task-trace-link"
+                href={getHoneycombTraceUrl(taskTrace, startTime, finishTime)}
+                onClick={() => {
+                  taskAnalytics.sendEvent({
+                    name: "Clicked metadata link",
+                    "link.type": "honeycomb trace link",
+                  });
+                }}
+                hideExternalIcon={false}
+              >
+                Honeycomb Trace
+              </StyledLink>
+              <StyledLink
+                data-cy="task-metrics-link"
+                href={getHoneycombSystemMetricsUrl(
+                  taskId,
+                  // @ts-expect-error: FIXME. This comment was added by an automated script.
+                  diskDevices,
+                  startTime,
+                  finishTime,
+                )}
+                onClick={() => {
+                  taskAnalytics.sendEvent({
+                    name: "Clicked metadata link",
+                    "link.type": "honeycomb metrics link",
+                  });
+                }}
+                hideExternalIcon={false}
+              >
+                Honeycomb System Metrics
+              </StyledLink>
+            </HoneycombLinkContainer>
+          </MetadataItem>
+        )}
+        {stepback && <Stepback taskId={taskId} />}
+      </MetadataCard>
+
+      {!isDisplayTask && (
+        <MetadataCard>
+          <MetadataTitle>Host Information</MetadataTitle>
+          {ami && (
+            <MetadataItem data-cy="task-metadata-ami">
+              <MetadataLabel>AMI:</MetadataLabel> {ami}
+            </MetadataItem>
+          )}
+          {!isContainerTask && distroId && (
+            <MetadataItem>
+              <MetadataLabel>Distro:</MetadataLabel>{" "}
+              <StyledRouterLink
+                data-cy="task-distro-link"
+                onClick={() =>
+                  taskAnalytics.sendEvent({
+                    name: "Clicked metadata link",
+                    "link.type": "distro link",
+                  })
+                }
+                to={getDistroSettingsRoute(distroId)}
+              >
+                {distroId}
+              </StyledRouterLink>
+            </MetadataItem>
+          )}
+          {!isContainerTask && hostId && (
+            <MetadataItem>
+              <MetadataLabel>Host:</MetadataLabel>{" "}
```

Comment:
[nit] What do you think about labeling this `ID:`? Since we sometimes ask users for a host ID it's probably familiar enough terminology